packages/cloud-cognitive/src/components/InlineEdit/InlineEdit.test.js

```diff
@@ -75,4 +76,197 @@ describe(componentName, () => {
+
+    // types some text
+    userEvent.keyboard('123');
+    expect(input).toHaveTextContent(`${startingValue}123`);
+
+    // testing {ctrl}z does not seem to work
+    // userEvent.type(input, '{ctrl}z');
+    // expect(input).toHaveTextContent(startingValue);
+  });
+
+  it('can start and cancel an edit using buttons', async () => {
+    const startingValue = value;
+    const { container } = render(
+      <InlineEdit {...requiredProps} value={startingValue} />
+    );
+    const component = container.querySelector(`.${blockClass}`);
+
+    const input = screen.getByRole('textbox');
+
+    // starts edit
+    const edit = container.querySelector(`.${blockClass}__edit`);
+    userEvent.click(edit);
+    expect(component).toHaveClass(`${blockClass}--editing`);
+
+    // has edit buttons
+    let controls = screen.getAllByRole('button');
+    expect(controls).toHaveLength(2); // cancel and save
+
+    // click cancel check result reverts
+    userEvent.click(controls[0]);
+    expect(input).toHaveTextContent(startingValue);
+    expect(component).not.toHaveClass(`${blockClass}--editing`);
+  });
+
+  it('can start and save edit with buttons', async () => {
+    const startingValue = value;
+    const { fn } = jest;
+    const handleChange = fn();
+    const handleSave = fn();
+    const { container } = render(
+      <InlineEdit
+        {...requiredProps}
+        value={startingValue}
+        onChange={handleChange}
+        onSave={handleSave}
+      />
+    );
+    const component = container.querySelector(`.${blockClass}`);
+
+    const input = screen.getByRole('textbox');
+
+    // starts edit
+    const edit = container.querySelector(`.${blockClass}__edit`);
+    userEvent.click(edit);
+    expect(component).toHaveClass(`${blockClass}--editing`);
+
+    // types some text
+    userEvent.type(input, 'ABC');
+    expect(input).toHaveTextContent(`${startingValue}ABC`);
+
+    // has edit buttons
+    let controls = screen.getAllByRole('button');
+    expect(controls).toHaveLength(2); // cancel and save
+
+    // click cancel check result reverts
+    userEvent.click(controls[1]);
+    expect(input).toHaveTextContent(startingValue);
+    expect(component).not.toHaveClass(`${blockClass}--editing`);
+    expect(input).toHaveTextContent(`${startingValue}ABC`);
+
+    expect(handleChange).toBeCalledTimes(3);
+    expect(handleSave).toBeCalled();
+  });
+
+  it('does not allow disabled edit', async () => {
+    const startingValue = value;
+
+    const { container } = render(
+      <InlineEdit {...requiredProps} value={startingValue} disabled={true} />
+    );
+    const component = container.querySelector(`.${blockClass}`);
+
+    // starts edit
+    const edit = container.querySelector(`.${blockClass}__edit`);
+    userEvent.click(edit);
+    expect(component).not.toHaveClass(`${blockClass}--editing`);
+  });
+
+  it('Clicking control but not on edit or input starts edit', async () => {
```

Comment:
You don't need the async as you're not doing any awaits.

```suggestion
  it('Clicking control but not on edit or input starts edit', () => {
```